### ISSUE TITLE
Info alerts in accordions are hidden to AT

### ISSUE THREAD

**Issue by scottqueen-bixal** [GSA/px-benefit-finder#1183]
### What's expected

"aria-hidden elements are not focusable nor contain focusable elements"

### What happened

info alerts in accordions should have a state of `aria-hidden="false"`

### How to replicate

- navigate to results
- inspect element , or
- run axeDevTools

![Image](https://github.com/GSA/px-benefit-finder/assets/37077057/d028c17d-0c97-4f83-9143-1d347c9d1f12)

**Comment by scottqueen-bixal** [comment]
We suspected we might have issues with too much verbosity in announcements to AT users with the alert component exposed,

here is a recording,


https://github.com/GSA/px-benefit-finder/assets/37077057/06cf4ca5-f195-4196-9ccc-0e502296beda

Based on this, we may want to have design update the component so we are not using alerts here - @angelina-smith can you confirm that this will be a more accessible user expereince?

**Comment by angelina-smith** [comment]
@scottqueen-bixal I agree that we should use not be using alerts within the accordions. There are probably a few different ways to update the design and text placement for better AT experience.

**Comment by ErnestoFernandezAlvarez** [comment]
Hello @scottqueen-bixal @angelina-smith @diegocob 

The message alert in the most updated design (within accordions) reads: "Visit the agency for full eligibility and requirements"
Please let me know if we need to further revise or change it.
Thank you!

**Comment by scottqueen-bixal** [comment]
@ErnestoFernandezAlvarez this is an a11y barrier. Please connect with @angelina-smith for more details on why we won't be able to continue using the alert inside of accordions and will need to alter the design accordingly.

**Comment by ErnestoFernandezAlvarez** [comment]
Thanks @scottqueen-bixal !!
@angelina-smith I will reach out on Team to discuss options, I'd love to hear your recommendations for these:)

**Comment by scottqueen-bixal** [comment]
Closing in lieu of considerations for replacement that are taking place on this ticket -> https://github.com/GSA/px-benefit-finder/issues/1196
